apps/blockchain/scripts/generate_state_tests.exs

```diff
@@ -0,0 +1,2571 @@
+defmodule GenerateStateTests do
+  alias MerklePatriciaTree.Trie
+  alias Blockchain.{Account, Transaction}
+  alias Blockchain.Interface.AccountInterface
+
+  use EthCommonTest.Harness
+  # use ExUnit.Case, async: true
+
+  @ethereum_common_tests_path "../../ethereum_common_tests"
```

Comment:
I think this could be replaced with `EthCommonTest.Helpers.ethereum_common_tests_path` (we'd have to make it a public function). That way we only have that relative url in a single place. What do you think?